src/compiler/transformers/module/module.ts

```diff
@@ -1774,28 +1801,28 @@ namespace ts {
                 && !isDeclarationNameOfEnumOrNamespace(node.operand)) {

```

Comment:
are these conditions still necessary now that this switched from substitution to transforming? If so, why are they not needed when transforming destructuring assignments?